### ISSUE TITLE
PS Remote Play no longer works after recent Sony update

### ISSUE THREAD

**Issue by Lawrencechl27** [Valkirie/HandheldCompanion#389]
After updated remote play can not be used normally，remote play can detected buy not function .

**Comment by CasperH2O** [comment]
Hey @Lawrencechl27 I do not understand what you mean. You updated from which version to which version? What is not working?

**Comment by higundamhws** [comment]
I think he means PS remote play can't control after it updated. It's same a my ayaneo geek issue

**Comment by Lawrencechl27** [comment]
> I think he means PS remote play can't control after it updated. It's same as my ayaneo geek issue to remote PS5.
> Hope it can be fixed before FF16 release.

Yes . Thank you for explain .

**Comment by CasperH2O** [comment]
@higundamhws @Lawrencechl27 after doing some digging and plenty more people reporting in, it seems Sony broke the virtual controller suppor in the recent PS Remote Play update. 

This is effecting pretty much any software that uses this, HC, Motion Assistant and even reWASD. 

So... good news is that it's not HC or anything we changed recently. The only "work around" I found was to use an older version of remote play and don't update, but that only gives you LAN access, no online gaming.

**Comment by IstanbulRed** [comment]
Hey Casper, thanks for looking into this. Its a shame this happened, it the easy DS4 virtual controller worked superbly with the official remote play app. It was excellent. I see the reWASD people have put out a fix, so hopefully you might be able to do the same at some point. Thank you again for this great piece of software - it takes my SteamDeck to the next level!

**Comment by BriaN890501** [comment]
> Hey Casper, thanks for looking into this. Its a shame this happened, it the easy DS4 virtual controller worked superbly with the official remote play app. It was excellent. I see the reWASD people have put out a fix, so hopefully you might be able to do the same at some point. Thank you again for this great piece of software - it takes my SteamDeck to the next level!

reWASD's solution is to create a virtual device under the physical USB hub, which emulates a real device for PS Remote Play.
Handheld Companion has been an amazing app for my steam deck running windows, looking forward to the remote play fix!

**Comment by MateusTorquato** [comment]
So, 9 months has past since last comment, so, it is still not possible to bypass it yet somehow? Just curious

**Comment by MSeys** [comment]
I'm going to assume not. There are alternative apps to do Remote Play, like Chiaki.

**Comment by MateusTorquato** [comment]
> I'm going to assume not. There are alternative apps to do Remote Play, like Chiaki.

This is unfortunate :/ I was trying to find a way to play on Playstation Plus app, and I think Chiaki is exclusive to remote play. Do you know some alternative to it that works on ps plus app?

**Comment by MSeys** [comment]
> > I'm going to assume not. There are alternative apps to do Remote Play, like Chiaki.
> 
> This is unfortunate :/ I was trying to find a way to play on Playstation Plus app, and I think Chiaki is exclusive to remote play. Do you know some alternative to it that works on ps plus app?

There is no way to do PS+ Cloud streaming using Remote Play. The official app does NOT support it either. All apps (first party and third party) use the same protocol to connect to the device and are under similar restrictions.

For PS+ Streaming, use the PC app itself.

**Comment by MateusTorquato** [comment]
Yeah I know, I'm using the ps+ official app for pc, but it sucks that I can't use any other controller, it only works with dualsense, but it seems there's really no easy way to bypass it for now.

**Comment by MSeys** [comment]
For PS5 games cloud streaming, there's no workaround indeed until we can emulate DualSense controllers. Maybe there are other applications that can do that instead.
